### PR TITLE
override media connector increment speed

### PR DESCRIPTION
**Pull request by sanky10987**
override media connector increment speed for fast forwarding with headset

### CODE REVIEW

File: app/src/main/java/com/koalatea/sedaily/feature/player/AudioService.kt [hunk 1]
```diff
@@ -201,7 +201,10 @@ class AudioService : LifecycleService() {
                             .build()
                 }
             })
-
+            // override media connector increment speed
```
Comment (MostafaGazar) [inline app/src/main/java/com/koalatea/sedaily/feature/player/AudioService.kt]: Please add a new line before the comment

File: app/src/main/java/com/koalatea/sedaily/feature/player/AudioService.kt [hunk 2]
```diff
@@ -201,7 +201,10 @@ class AudioService : LifecycleService() {
...
+            val incrementMs = resources.getInteger(R.integer.increment_ms)
+            setFastForwardIncrementMs(incrementMs)
+            setRewindIncrementMs(incrementMs)
             setPlayer(exoPlayer)
```
Comment (MostafaGazar) [inline app/src/main/java/com/koalatea/sedaily/feature/player/AudioService.kt:209]: Before add a new line before this line to keep each logical block of code separate

File: app/src/main/java/com/koalatea/sedaily/feature/player/AudioService.kt (hunk 1 above)
Comment (MostafaGazar) [inline app/src/main/java/com/koalatea/sedaily/feature/player/AudioService.kt]: Can you add a more descriptive comment, or feel free to get rid of

**Review by MostafaGazar — CHANGES_REQUESTED** [review]
Thanks for your contribution, I added a few comments regarding the code styling

**Comment by MostafaGazar** [comment]
This should fix https://github.com/SoftwareEngineeringDaily/software-engineering-daily-android/issues/35

File: app/src/main/java/com/koalatea/sedaily/feature/player/AudioService.kt (hunk 1 above)
Comment (sanky10987) [reply app/src/main/java/com/koalatea/sedaily/feature/player/AudioService.kt]: Added New line

File: app/src/main/java/com/koalatea/sedaily/feature/player/AudioService.kt (hunk 2 above)
Comment (sanky10987) [reply app/src/main/java/com/koalatea/sedaily/feature/player/AudioService.kt:209]: Added new line

File: app/src/main/java/com/koalatea/sedaily/feature/player/AudioService.kt (hunk 1 above)
Comment (sanky10987) [reply app/src/main/java/com/koalatea/sedaily/feature/player/AudioService.kt]: Got rid of comment completely

**Review by sanky10987 — COMMENTED** [review]
Fixed the review comments

**Pull request merged**